features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FilterViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.agilefreaks.freaks_catalog.features.freaks
+
+import androidx.lifecycle.ViewModel
+
+class FilterViewModel : ViewModel() {
+    companion object {
```

Comment:
There seems to be some guidance about where we should put out companion objects in a class. Did you read about this? Check this out and let us know what you think about this. 
https://kotlinlang.org/docs/coding-conventions.html#class-layout

Reply:
I haven't read about this before. I think it's nice to have a set of rules in order to keep things clean, organized and easier to read.